temporal-sdk/src/main/java/io/temporal/nexus/OperationHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2022 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ * Copyright (C) 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this material except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.temporal.nexus;
+
+import io.nexusrpc.handler.*;
+import io.temporal.common.Experimental;
+import io.temporal.internal.nexus.CurrentNexusOperationContext;
+import io.temporal.internal.nexus.NexusOperationContextImpl;
+
+/**
+ * Helper to create {@link io.nexusrpc.handler.OperationHandler} instances that take a {@link
+ * io.temporal.client.WorkflowClient}.
+ */
+@Experimental
+public final class OperationHandler {
```

Comment:
I wonder if we need a slightly different name to disambiguate if both this and the `io.nexusrpc` equivalent are imported. How about `WorkflowClientOperationHandler` and `SynchronousWorkflowClientOperationFunction`?